invenio_records_marc21/services/services.py

```diff
@@ -85,6 +90,47 @@ def update_draft(
         data = self._create_data(identity, data, metadata, access)
         return super().update_draft(id_, identity, data, revision_id)
 
+    def _lift_embargo_from(self, record):
+        """Lifts embargo from record or draft."""
+        if not record.access.embargo.lift():
+            raise EmbargoNotLiftedError(record["id"])
+        record.access.protection.metadata = "public"
+        record.access.protection.files = "public"
+
+    def _draft_access_field_was_modified(self, draft, record):
```

Comment:
the name does not suggest, that the return type is a bool. i would prefer "_is_draft_access_field_modified"

Reply:
Ok